src/components/pages/order/Main/admin/PanelContent.js

```diff
@@ -0,0 +1,29 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import { theme } from "../../../../../theme";
+import OrderContext from "../../../../../context/OrderContext";
+import { tabsConfig } from "./tabsConfig";
+
+export default function PanelContent() {
+  //state
+  const { currentTabSelected } = useContext(OrderContext);
+  //comportement
+  const tabs = tabsConfig;
+  const tabSelected = tabs.find((tab) => currentTabSelected === tab.index);
```

Comment:
```suggestion
  const tabSelected = getTabsSelected(currentTabSelected)
```
Créé une fonction à part pour que ce soit plus propre. Tu peux ranger cette fonction dans tabsConfig.js

Reply:
J'suis pas fan, dans l'immédiat je ne l'utilise qu'ici, là faut que je le mette dans un autre fichier, que je rajoute une ligne d'import et que je rappel une fonction. Pour économiser une ligne tu me proposes d'en créer 3 ? Enfin je le vois comme ça peut-être que je suis dans l'erreur